CHANGELOG.md

```diff
@@ -2,7 +2,40 @@
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/).
 
-## [Unreleased](https://github.com/trailofbits/manticore/compare/0.1.4...HEAD)
+## [Unreleased](https://github.com/trailofbits/manticore/compare/0.1.5...HEAD)
+
+## 0.1.5 - 2017-09-15
+
+Thanks to our external contributors to this release!
+
+- [johnfxgalea](https://github.com/johnfxgalea)
```

Comment:
idea: new contributor shoutout style, i wasn't sure how to phrase the Linux syscalls line